### ISSUE TITLE
Colorized Output not working (1.0.8)

### ISSUE THREAD

**Issue by pnd280** [loganch/AutoIt-VSCode#100]
I was using [Output Colorizer](https://marketplace.visualstudio.com/items?itemName=IBM.output-colorizer). But after update to 1.0.8, the extension doesn't seem to change the output color (already uninstalled the previous one and also tried to disable all other extensions). 
Do i need to do some specific settings for this feature to work on 1.0.8?

Current environment: VSCode 1.68.1 (Windows)

**Comment by Danp2** [comment]
Likely due to changes made in https://github.com/loganch/AutoIt-VSCode/pull/80

**Comment by pnd280** [comment]
Yes i mean that im having trouble using the inbuilt colorizer that you have mentioned above even after i uninstalled [Output Colorizer](https://marketplace.visualstudio.com/items?itemName=IBM.output-colorizer).
![image](https://user-images.githubusercontent.com/13845731/175820012-515700cc-fec0-4a25-8ee9-d0dc15ab0556.png)

**Comment by Danp2** [comment]
> Do i need to do some specific settings for this feature to work on 1.0.8

Not as far as I can recall. It's working fine for me, but maybe that's because I made the original mod. @loganch Does the following need to be added to `package.json` in the new section for `vscode-autoit-output` language?

`
        "configuration": "syntaxes\vscode-autoit-output.tmLanguage.json"
`

**Comment by Danp2** [comment]
I checked the .vsix file and this file is present there. I also looked in `%USERPROFILE%\.vscode\extensions\damien.autoit-1.0.8\syntaxes` and it is there as well.

@pnd280 You may want to check the logs (`ctrl+shift+p` -> `Show Logs...` -> `Extension Host`) to see if there are any clues.

**Comment by pnd280** [comment]
```
[2022-06-27 12:04:58.538] [exthost] [info] ExtensionService#_doActivateExtension Damien.autoit, startup: false, activationEvent: 'onLanguage:autoit'
```

I couldnt find any logs related to the extension other than this

**Comment by vanowm** [comment]
Fixed in https://github.com/loganch/AutoIt-VSCode/commit/dfef5198c0d67a936a92d87f2154e40642374abd
